src/PrestaShopBundle/Form/Admin/Sell/CustomerService/MerchandiseReturn/OrderReturnFormDataProvider.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\CustomerService\OrderReturn;
+
+use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Form\FormDataProviderInterface;
+
+/**
+ * Provides order return options form with data
+ */
+final class OrderReturnFormDataProvider implements FormDataProviderInterface
```

Comment:
Mmm this one is weird it's confusing with the `PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataProvider\OrderReturnFormDataProvider` and besides I don't see any uses of this service anywhere

It more looks more like a provider for a Shop configuration form

If you need a service to change the options of the `OrderReturnType` you can instead rely on the `FormOptionsProviderInterface` The only use of this interface for now is this one https://github.com/PrestaShop/PrestaShop/blob/7122c895b442ab8c4bb83aee3a38fbb9ece1c8b5/src/Core/Form/IdentifiableObject/OptionProvider/CombinationFormOptionsProvider.php